zcash_client_sqlite/src/wallet/scanning.rs

```diff
@@ -1412,11 +1430,14 @@ pub(crate) mod tests {
         // We've crossed a subtree boundary, but only in one pool. We still only have one scanned
         // note but in the pool where we crossed the subtree boundary we have two shards worth of
         // notes to scan.
+        let expected_denom: u64 = 2823;
```

Comment:
This one I was failing to calculate because the existing test was producing a broken shard tree state:
```
SELECT shard_index, subtree_end_height FROM sapling_tree_shards
[
    (0, Some(100066)),
    (1, None),
    (2, Some(100646)),
]
```

Reply:
That's interesting, because in the absence of downloading subtree roots this *is* a valid state for the tree.